src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -653,15 +653,16 @@ private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSy
             int variableCount = variableList.Count;
             if (variableCount == 1)
             {
-                return BindVariableDeclaration(kind, isVar, variableList[0], typeSyntax, declType, alias, diagnostics, node);
+                return BindVariableDeclaration(kind, isVar, variableList[0], typeSyntax, declType, alias, diagnostics, true, node);
```

Comment:
>true [](start = 119, length = 4)

Please use a parameter keyword syntax whenever you use a literal for an argument.